fix: Unnecessary Spaces And Grammar (#1795)
EddieHubCommunity/support: youtube/generic.md

## YouTube Channel

> Don’t forget to support my channel by subscribing it! It’s free: [youtube.com/eddiejaoude](https://youtube.com/eddiejaoude?sub_confirmation=1)

Reviewer: afaik Markdown does **not** work in youtube description.
The entire point of this file is so that Eddie can copy-and-paste the text onto the description section.
```suggestion
Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```
It would be great if we could avoid using specific-markdown(like hyperlinks), wherever possible.

**Make sure to tell your friends about it, so that they also learn something new!**

## YouTube Shorts

> I have another YouTube channel for short clips, please **take a moment subscribing it: [youtube.com/eddiejaoudetv](https://youtube.com/eddiejaoudetv?sub_confirmation=1)**

Reviewer: ```suggestion
I have another YouTube channel for short clips, please **take a moment subscribing it: https://youtube.com/eddiejaoudetv?sub_confirmation=1**
```

## Join Our Community

> Join our inclusive open source community EddieHub now for **free** at [eddiehub.org](https://www.eddiehub.org)

Reviewer: ```suggestion
Join our inclusive open source community EddieHub now for **free** at https://www.eddiehub.org
```

 
---

> Make sure to checkout my website and sign up to our community **newsletter**: [eddiejaoude.io](https://www.eddiejaoude.io) for my live stream + video schedule and community calls

Reviewer: ```suggestion
Make sure to checkout my website and sign up to our community **newsletter**: https://www.eddiejaoude.io for my live stream + video schedule and community calls
```

Join Our **Community GitHub Organization**:
 

> - Go to [EddieHubCommunity/support/issues/new/choose](https://github.com/EddieHubCommunity/support/issues/new/choose) 

Reviewer: ```suggestion
- Go to https://github.com/EddieHubCommunity/support/issues/new/choose 
```

> - Go to [EddieHubCommunity/support/issues/new/choose](https://github.com/EddieHubCommunity/support/issues/new/choose) 

Reviewer: These unordered lists don't render properly when copied and pasted. Maybe we can edit and make it look like-
```suggestion
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
\- Choose The Issue Type Of **Invite Me To The Organization**
```
this renders- 
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
\- Choose The Issue Type Of **Invite Me To The Organization**

- Choose The Issue Type Of **Invite Me To The Organization**
- Open The Issue
- You get an invitation to the community by our GitHub Action 🎉

---

### GitHub Sponsors

> - Stephen Mount: [github.com/stemount](https://github.com/stemount)

Reviewer: ```suggestion
- Stephen Mount: https://github.com/stemount
```

> - Nicholas Carrigan: [github.com/nhcarrigan](https://github.com/nhcarrigan)

Reviewer: ```suggestion
- Nicholas Carrigan: https://github.com/nhcarrigan
```

> - Santosh Yadav: [github.com/santoshyadavdev](https://github.com/santoshyadavdev)

Reviewer: ```suggestion
- Santosh Yadav: https://github.com/santoshyadavdev
```

> - Andrew Cunliffe: [github.com/andrew-cunliffe](https://github.com/andrew-cunliffe)

Reviewer: ```suggestion
- Andrew Cunliffe: https://github.com/andrew-cunliffe
```

> - Alexander Stoichkov: [github.com/SashoStoichkov](https://github.com/SashoStoichkov)

Reviewer: ```suggestion
- Alexander Stoichkov: https://github.com/SashoStoichkov
```

> - Allan Regush: [github.com/AllanRegush](https://github.com/AllanRegush)

Reviewer: ```suggestion
- Allan Regush: https://github.com/AllanRegush
```

 
---
 
## Social Media
Follow on socials for behind the scenes footage and more.

> - Website: [eddiejaoude.io](https://www.eddiejaoude.io) 

Reviewer: ```suggestion
- Website: https://www.eddiejaoude.io 
```

> - GitHub: [github.com/eddiejoude](https://github.com/eddiejaoude)

Reviewer: ```suggestion
- GitHub: https://github.com/eddiejaoude
```

> - YouTube: [youtube.com/eddiejoude](https://youtube.com/eddiejaoude)

Reviewer: ```suggestion
- YouTube: https://youtube.com/eddiejaoude
```

> - Twitter: [twitter.com/eddiejaoude](https://twitter.com/eddiejaoude)

Reviewer: ```suggestion
- Twitter: https://twitter.com/eddiejaoude
```

> - Instagram: [instagram.com/eddiejaoude](https://instagram.com/eddiejaoude)

Reviewer: ```suggestion
- Instagram: https://instagram.com/eddiejaoude
```

> - LinkedIn: [linkedin.com/in/eddiejaoude](https://linkedin.com/in/eddiejaoude)

Reviewer: ```suggestion
- LinkedIn: https://linkedin.com/in/eddiejaoude
```

> - GitHub Community Org: [github.com/EddieHubCommunity](https://github.com/EddieHubCommunity)

Reviewer: ```suggestion
- GitHub Community Org: https://github.com/EddieHubCommunity
```

## Further

> Join Our Conversation At Discord: [discord.eddiehub.org](https://discord.eddiehub.org/)

Reviewer: ```suggestion
Join Our Conversation At Discord: https://discord.eddiehub.org/
```

## YouTube Channel
Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
**Make sure to tell your friends about it, so that they also learn something new!**

## YouTube Shorts
I have another YouTube channel for short clips, please **take a moment subscribing it: https://youtube.com/eddiejaoudetv?sub_confirmation=1**

## Join Our Community
Join our inclusive open source community EddieHub now for **free** at https://www.eddiehub.org
 
---

Make sure to checkout my website and sign up to our community **newsletter**: https://www.eddiejaoude.io for my live stream + video schedule and community calls

Join Our **Community GitHub Organization**:
 

> - Go to https://github.com/EddieHubCommunity/support/issues/new/choose

Reviewer: I think we would want these to appear as bullet points on youtube too. (normal bullets in markdown get copied as normal text, but we could ignore this line to get a copy-able list)
```suggestion
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
```

> - Choose The Issue Type Of **Invite Me To The Organization**

Reviewer: ```suggestion
\- Choose The Issue Type Of **Invite Me To The Organization**
```

> - Choose The Issue Type Of **Invite Me To The Organization**

Reviewer: ```suggestion
\- Choose The Issue Type Of **Invite Me To The Organization**
```

> - Open The Issue

Reviewer: ```suggestion
\- Open The Issue
```

> - You get an invitation to the community by our GitHub Action 🎉

Reviewer: ```suggestion
\- You get an invitation to the community by our GitHub Action 🎉
```

---

### GitHub Sponsors
- Stephen Mount: https://github.com/stemount
- Nicholas Carrigan: https://github.com/nhcarrigan
- Santosh Yadav: https://github.com/santoshyadavdev
- Andrew Cunliffe: https://github.com/andrew-cunliffe
- Alexander Stoichkov: https://github.com/SashoStoichkov
- Allan Regush: https://github.com/AllanRegush
 
---
 
## Social Media
Follow on socials for behind the scenes footage and more.

- Website: https://www.eddiejaoude.io 
- GitHub: https://github.com/eddiejaoude
- YouTube: https://youtube.com/eddiejaoude
- Twitter: https://twitter.com/eddiejaoude
- Instagram: https://instagram.com/eddiejaoude
- LinkedIn: https://linkedin.com/in/eddiejaoude
- GitHub Community Org: https://github.com/EddieHubCommunity

## Further

> Join Our Conversation At Discord: https://discord.eddiehub.org/

Reviewer 2: Hey! The discord link isn't working, I have checked with the working link it contains **http** I wonder why it happens
have a look at it once [http://discord.eddiehub.org/](http://discord.eddiehub.org/)

Reviewer: oh wupps slipped past me - http:///discord.eddiehub.org/ would be the correct link

Reviewer 2: @Vyvy-vi may I know why it is not working for https 🤔

Reviewer 3: > @Vyvy-vi may I know why it is not working for https 🤔 

Why?

Reviewer: The discord link sub-domain is a URL-forward, instead of a normal CNAME or A level record(since discord's invite is case sensitive).
And thus, I think it doesn't get the SSL cert. from the root domain(if Eddie's provider supports that thing)
But, if Eddie's domain provider doesn't support inheriting SSL certs to sub-domains, then this just ends up as http.

There might be a way to make this work with https tho, with something like cloudfare acting as an intermediary

Reviewer 4: It's a DNS level redirect, which doesn't support HTTPS. You can't use a CNAME or A record because those don't accept URL paths. 

It would be possible, however, to spin up a small web server to handle the redirect and maintain HTTPS.

## YouTube Channel
Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1

MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!

## YouTube Shorts
I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1

## Newsletter

Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls

## Join Our Community
Join our inclusive open source community EddieHub now for FREE at https://www.eddiehub.org
 
\1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose
\2. Choose The Issue Type Of Invite Me To The Organization
\3. Choose The Issue Type Of Invite Me To The Organization
\4. Open The Issue
\5. You get an invitation to the community by our GitHub Action 🎉

---

## Community Sponsors

Stephen Mount http://github.com/stemount
Nicholas Carrigan https://github.com/nhcarrigan

> Santosh Yadav https://github.com/santoshyadavdev

Reviewer: while copy pasting, all these bullets get added to the same line.
A solution might be to split them into new lines -
```suggestion
\- Santosh Yadav https://github.com/santoshyadavdev
```

Reviewer: somewhat like this-
```
\- Santosh Yadav https://github.com/santoshyadavdev
\- Stephen Mount http://github.com/stemount
```
\- Santosh Yadav https://github.com/santoshyadavdev
\- Stephen Mount http://github.com/stemount

Andrew Cunliffe https://github.com/andrew-cunliffe
Alexander Stoichkov https://github.com/SashoStoichkov
Allan Regush https://github.com/AllanRegush

---

## Service I Use and Recommend

> Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too

Reviewer: ```suggestion
Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too
```

> NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4

Reviewer: while copy pasting, all these bullets get added to the same line.
A solution might be to split them into new lines -

```suggestion
\- NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4
```

This would prevent this to get weirdly formatted while copy-pasting from the markdown file:

![Screenshot 2021-06-25 at 1 26 12 PM](https://user-images.githubusercontent.com/62864373/123390917-1309af80-d5b9-11eb-979a-d413a92be8b5.png)

## YouTube Channel
Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1

MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!

## YouTube Shorts
I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1

## Newsletter

Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls

## Join Our Community
Join our inclusive open source community EddieHub now for FREE at https://www.eddiehub.org
 
1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose

> 2. Choose The Issue Type Of Invite Me To The Organization

Reviewer: points 2 and 3 are duplicates

> ## YouTube Channel

Reviewer 5: ```suggestion
# YouTube Channel
```

Don't forget to support my channel by subscribing to it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1

MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY CAN ALSO LEARN SOMETHING NEW!

> ## YouTube Shorts

Reviewer 5: ```suggestion
## YouTube Shorts

```

I have another YouTube channel for short clips, please take a moment subscribing to it: https://youtube.com/eddiejaoudetv?sub_confirmation=1

## Newsletter

Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls

> ## Join Our Community

Reviewer 5: ```suggestion
## Join Our Community

```

## YouTube Channel

> Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1

Reviewer 5: Hmm, shouldn't this be `'` instead of `’`?
```suggestion
Don't forget to support my channel by subscribing to it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```

> MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!

Reviewer: ```suggestion
MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY CAN ALSO LEARN SOMETHING NEW!
```
I think maybe re-phrasing like that might sound better?

## YouTube Shorts

> I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1

Reviewer: ```suggestion
I have another YouTube channel for short clips, please take a moment subscribing to it: https://youtube.com/eddiejaoudetv?sub_confirmation=1
```
I think the capitals might not look very natural. (Non-violent-communication in phrasing like these might look better). Thoughts, @eddiejaoude ?

Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
## Join Our Community
Join our inclusive open source community EddieHub now for FREE at https://www.eddiehub.org
 
1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose

## SERVICES I USE AND RECOMMEND
2. Choose The Issue Type Of Invite Me To The Organization

Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
3. Open The Issue

- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
- Awesome backup platform, BackBlaze http://links.eddiejaoude.io/t/4rqTZjLbwmCp
4. You get an invitation to the community by our GitHub Action 🎉

## COMMUNITY SPONSORS
---

## Community Sponsors

A BIG thank you to my GitHub Sponsors:

> ***

Reviewer: I'm not sure, but I think those Asterisks were there since Stephen and Nicholas are Eddie's biggest sponsors, or something like that. _Not sure_

Author: I also kept the difference but I was asked by ig you or @kaiwalyakoparkar to remove the diff

Author: ig maybe we can ask @eddiejaoude for to keep the diff/not keep the diff

> Andrew Cunliffe https://github.com/andrew-cunliffe
> 
> Alexander Stoichkov https://github.com/SashoStoichkov
> 
> Allan Regush https://github.com/AllanRegush

Reviewer 5: ```suggestion
[Andrew Cunliffe](https://github.com/andrew-cunliffe)\
[Alexander Stoichkov](https://github.com/SashoStoichkov)\
[Allan Regush](https://github.com/AllanRegush)
```

That would make it look like

[Andrew Cunliffe](https://github.com/andrew-cunliffe)\
[Alexander Stoichkov](https://github.com/SashoStoichkov)\
[Allan Regush](https://github.com/AllanRegush)

Author: Markdown is not supported in yt vid descriptions, sorry

Reviewer 5: Oh, I didn't know this was for YouTube videos 😅 Sorry.

> ## Service I Use and Recommend

Reviewer 5: ```suggestion
## Services I use and recommend
```

> Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too

Reviewer 5: ```suggestion
Some of these might be an affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount, too.
```

\- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
\- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
\- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
\- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
\- Awesome backup platform, BackBlaze http://links.eddiejaoude.io/t/4rqTZjLbwmCp

---
 

> ## Find me on social media

Reviewer: I think I'm nitpicking here, but Eddie usually refers to them as `socials` 😅:
```suggestion
## Find me on socials
```

> Website: https://www.eddiejaoude.io 
> 
> GitHub: https://github.com/eddiejaoude
> 
> YouTube: https://youtube.com/eddiejaoude
> 
> Twitter: https://twitter.com/eddiejaoude

Reviewer 5: ```suggestion
Website: https://www.eddiejaoude.io\
GitHub: https://github.com/eddiejaoude\
YouTube: https://youtube.com/eddiejaoude\
Twitter: https://twitter.com/eddiejaoude\
```

Author: Tried but unacceptedly, he whole thing appears on one line

> 

Reviewer 5: ```suggestion
```

Instagram: https://instagram.com/eddiejaoude

LinkedIn: https://linkedin.com/in/eddiejaoude

> GitHub Community Org: https://github.com/EddieHubCommunity

Reviewer 5: ```suggestion
GitHub Community Organization link: https://github.com/EddieHubCommunity
```

> ## Further

Reviewer 5: Maybe `miscellaneous` would be better instead?
```suggestion
## Miscellaneous
```

Author: Futher seems a little more friendly to me. Miscellaneous will be a little to much formal context to the actual text inside

Reviewer 5: What about `more information`? Or `extras` or something like that? I feel `further` doesn't fit well there.

Author: > What about `more information`? Or `extras` or something like that? I feel `further` doesn't fit well there.

These words too aren't a fit here. But maybe **Meet our Community** will work?

Reviewer 5: That sounds great! 😄

Reviewer: Essentially, the Heading isn't really necessary
You could state it without a heading

> Join Our Conversation At Discord: http://discord.eddiehub.org/

Reviewer: ```suggestion
Join the Conversation at Discord: http://discord.eddiehub.org/
```
I think this re-phrasing might be grammatically neater. Thoughts?

> # YouTube Channel

Reviewer 5: ```suggestion
# YouTube Channel

```

## Service I Use and Recommend

> Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too

Reviewer 6: This sounds like a disclaimer - are we sure we can leave this out?

Reviewer 7: I think most people write something above their affiliate links, to avoid any confusion
